cmd/miniooni/main.go

```diff
@@ -210,11 +208,13 @@ func main() {
 	}
 
 	if !globalOptions.noBouncer {
+		log.Info("Start lookup OONI backends")
```

Comment:
I suggest `Looking up OONI backends` because you don't print `End lookup...` (and I'd not like this, since it would make the output take more vertical space than required.)